src/Symfony/Component/JsonStreamer/Tests/Fixtures/Model/DummyWithGenerics.php

```diff
@@ -8,7 +8,7 @@
 class DummyWithGenerics
 {
     /**
-     * @var array<int, T>
+     * @var list<T>
```

Comment:
Should we add a new fixture instead to keep testing with what it was before?

Reply:
`array<foo, bar>` are already tested in generators related tests, so I don't think this needs an additional one (for information, I updated that PHPDoc to simplify a bit the resulting JSON).